<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIgesturerecognitionsave</class>
 <widget class="QWidget" name="GUIgesturerecognitionsave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Gesture Recognition with HMM</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3290</width>
     <height>961</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Save gestures to file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textFilename">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max Window Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelControl">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Saving progress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Annotation Tool</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>315</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open a Json file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Frames</string>
    </property>
    <widget class="QLabel" name="labelFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="sliderFrame">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>721</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>373</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frame Slider</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarOpening">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Annotation</string>
    </property>
    <widget class="QLineEdit" name="lineBegin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEnd">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Begin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSubmit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>55</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSave">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSaveState">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonReset">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Frame Number</string>
    </property>
    <widget class="QLabel" name="labelNFrame">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFwd">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTotFrame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>112</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>620</y>
      <width>521</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Submissions</string>
    </property>
    <widget class="QTextBrowser" name="textBrowserSub">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>501</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>315</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Opening progress</string>
    </property>
   </widget>
  </widget>
  <action name="actionProject">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionKk">
   <property name="text">
    <string>kk</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
